src/wallet/rpcwallet.cpp

```diff
@@ -2117,6 +2117,77 @@ static UniValue listaccounts(const JSONRPCRequest& request)
     return ret;
 }
 
+static UniValue listsincetx(const JSONRPCRequest& request)
+{
+    CWallet* const pwallet = GetWalletForJSONRPCRequest(request);
+    if (!EnsureWalletIsAvailable(pwallet, request.fHelp)) {
+        return NullUniValue;
+    }
+
+    if (request.fHelp || request.params.size() < 1 || request.params.size() > 2)
+        throw std::runtime_error(
+            "listsincetx \"txid\" (polltime)\n"
+            "\nGet all transactions since a specific transaction [txid].\n"
+            "This call can be used as longpoll notification channel if follow-up transactions from the latest transaction-id are requested and [polltime] is greater then zero.\n"
+            "\nArguments:\n"
+            "1. \"txid\"               (string, required) The txid to list transactions since\n"
+            "1. \"polltime\"           (numeric, optional) If greater then zero, wait with a timeout of <polltime> seconds for new transactions. It will response immediately when new transactions arrive (default is 0)\n"
```

Comment:
greater than zero

Reply:
I don't understand that comment.

Reply:
s/then/than: `If greater than zero, wait with ...`